packages/expo-video/ios/VideoManager.swift

```diff
@@ -10,15 +10,26 @@ import ExpoModulesCore
 class VideoManager {
   static var shared = VideoManager()
 
+  private static var managerQueue = DispatchQueue(label: "com.expo.video.manager.managerQueue")
   private var videoViews = NSHashTable<VideoView>.weakObjects()
   private var videoPlayers = NSHashTable<VideoPlayer>.weakObjects()
 
-  func register(videoPlayer: VideoPlayer) {
-    videoPlayers.add(videoPlayer)
+  func register(videoPlayer: VideoPlayer, queue: DispatchQueue? = nil) {
```

Comment:
```suggestion
  func register(videoPlayer: VideoPlayer) {
```
Let's remove it anyway. You can always add it in the future 😅